packages/console/src/pages/Profile/components/FormCard/index.module.scss

```diff
@@ -0,0 +1,31 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  padding: _.unit(6) _.unit(8);
+  display: flex;
+  margin-top: _.unit(4);
+  background: var(--color-layer-1);
+  border-radius: 12px;
+}
+
+.title {
+  @include _.section-head-1;
+  width: 352px;
+  flex-shrink: 0;
+  color: var(--color-neutral-variant-60);
+}
+
+.content {
+  flex-grow: 1;
+}
+
+@media screen and (max-width: 1080px) {
```

Comment:
nit: we have defined consts for similar situations.

Reply:
New constant added `$screen-width-large: 1080px` in `_dimensions.scss`